src/transport_ofi.c

```diff
@@ -526,11 +528,7 @@ void shmem_transport_ofi_stx_allocate(shmem_transport_ctx_t *ctx)
 }
 
 #define OFI_MAJOR_VERSION 1
-#ifdef ENABLE_MR_RMA_EVENT
 #define OFI_MINOR_VERSION 5
-#else
-#define OFI_MINOR_VERSION 0
-#endif
 
```

Comment:
Trying to understand, so, we do not have MR type of RMA_EVENT any more? Or is it just the version got fixed for all MR types?

Reply:
My understanding is that we previously needed to use OFI v1.5 if we where enabling MR_RMA_EVENT (which was new to v1.5), but this PR moves us completely to v1.5.

Reply:
We now always require OFI v1.5 or newer API. Previously we only required it when RMA_EVENT support was enabled.